test/lib/ansible_test/_internal/manage_ci.py

```diff
@@ -296,12 +304,13 @@ def ssh(self, command, options=None, capture=False):
         if isinstance(command, list):
             command = ' '.join(cmd_quote(c) for c in command)
 
+        command = command if self.core_ci.platform in ['aix', 'ibmi'] else cmd_quote(command)
```

Comment:
It looks like this should be conditional on `self.become` rather than the platform:

```suggestion
        command = cmd_quote(command) if self.become else command
```